de.ifw_dresden.nagstamon.appdata.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
```

Comment:
The appdata file should contain the license comment and sent upstream :) not a blocker of course, but it's preferable.